zebra-network/src/meta_addr/tests/vectors.rs

```diff
@@ -1,6 +1,17 @@
 //! Test vectors for MetaAddr.
 
+use std::net::SocketAddr;
+
+use zebra_chain::serialization::{DateTime32, Duration32};
+
 use super::{super::MetaAddr, check};
+use crate::{constants::REACHABLE_PEER_DURATION, protocol::types::PeerServices};
+
+/// Margin of error for time-based tests.
+///
+/// This is a short duration to consider as error due to a test's execution time when comparing
+/// [`DateTime32`]s.
+const TEST_TIME_ERROR_MARGIN: Duration32 = Duration32::from_seconds(1);
```

Comment:
On heavily loaded VMs, we might want to allow 5-10 seconds here.

But let's see how it goes in CI.